scripts/aws/monitor.py

```diff
@@ -14,9 +14,12 @@
 # --lambda-timeout (The function execution time)
```

Comment:
Docs should indicate that this can currently only create a lambda that runs in a VPC, we don't need it to be flexible, but should note that fact.